### PR TITLE
added & abstracted signout button on navbar

### PR DESCRIPTION
**Pull request by jhylisa666**
* Create a component Layout in /components/layout/Layout.tsx and add all your styles for Layout (if any) in /components/layout/Layout.styles.ts
*  Inside Layout.tsx create a navbar -> see these docs https://mui.com/material-ui/react-app-bar/
 * Add the signout button onto the navbar (this should be the only thing on there) and verify signing out still works as expected
 * Wrap all the pages with the Layout component. After you do this, you should see the navbar on all of the pages
*  Make sure the nav bar is responsive and looks good from mobile -> desktop

Video --> 
https://user-images.githubusercontent.com/92692037/175548254-302371bb-2ea3-48e3-9018-c48b2668a38c.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.8.4",
```
Comment (jzhzhang66) [inline package.json:14]: yarn remove this package (b/c we don't need the hamburger icon or any other icons from this package right now)

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
HEY LISAAA<333 Thank you so much for working on this - you're doing great!! Super proud of ya for taking this on:)) You got the general idea - just left a few comments on how we can improve this even more! 

Also, sx is inline styling and as per our styling guide, if we want to style things we are using the styled utility from MUI. It seems like a lot of your sx styling isn't actually doing anything so for those we can remove them. But if you have a styling you intentionally do want and it is doing something, use the styled utility. You can see examples of how to do this in the files in /pageStyles folder!!!

**Review by karynayen — CHANGES_REQUESTED** [review]
OMG AMAZING WORK LISA!!! I also agree with judy about the color! maybe make the inner text white? and maybe make the button transparent with a white outline? or change the color so that it fits more with the themes? Whatever you think will look best!!

File: pages/emailSender.tsx [hunk 2]
```diff
@@ -157,6 +158,11 @@ const EmailSender: NextPage = () => {
 
   return (
     <ThemeProvider theme={theme}>
+       <Layout>
+      <Button variant="contained" color = "secondary">
```
[the remark this answers is not in the document]
Comment (jhylisa666) [reply pages/emailSender.tsx:162]: Woahhhhhh I seeeeee thankssss!!!!!!!

**Review by jzhzhang66 — APPROVED** [review]
BEAUT

**Review by karynayen — APPROVED** [review]
LOOKS GREAT!!

**Pull request merged**